db/migrate/20150902194024_create_memberships.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMemberships < ActiveRecord::Migration
+  def change
+    create_table :memberships do |t|
+      t.references :user
+      t.references :meal
+      t.string :role, default: 'participant'
```

Comment:
Let's keep the defaults out of the database migrations, especially since they're magic strings.